harvester/rr-dm3/src/main/java/dk/dbc/dataio/harvester/rr_dm3/RawRepoQueue.java

```diff
@@ -66,11 +58,7 @@ public RawRepoRecordHarvestTask peek() throws HarvesterException {
      */
     @Override
     public RawRepoRecordHarvestTask poll() throws HarvesterException {
-        try {
-            return peek();
-        } finally {
-            head = null;
-        }
+        return null;
```

Comment:
Kan den kaste en UnsupportedOperationException i stedet for bare at returnere null.

Og også meget gerne lige tilpasse Javadoc til virkeligheden.

Reply:
Klassen skal vist bare slettes